setup-env

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+USAGE=$(cat << 'END_OF_LINE'
+This script is used to configure a developement environment for this repo.
+
+It does the following:
+  - Verifies pyenv and pyenv-virtualenv are installed.
+  - Creates a Python virtual environment.
+  - Configures the activation of the virtual enviroment for the repo directory.
+  - Installs the requirements required for development.
+  - Installs git pre-commit hooks.
+  - Configures git upstream remote "lineage" repositories.
+
+usage: setup-env [--force] [--help] [virt_env_name]
```

Comment:
I feel like this should be first for convenience and to follow typical usage output practice. (I would also prefer usage to be capitalized).

Reply:
This is not BSD!  There are rules!  usage shall not be capitalized! And the revolution will not be televised! 
 
```console
❱ grep
usage: grep [-abcDEFGHhIiJLlmnOoqRSsUVvwxZ] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]

❱ sed 
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]

❱ awk
usage: awk [-F fs] [-v var=value] [-f progfile | 'prog'] [file ...]
```

That said... you are correct.  We should capitalize it because that is what we did with all our other tools in the dev guide:  https://github.com/cisagov/development-guide/tree/develop/project_setup/scripts

But I disagree that that "Usage" line should be first.  It is always at the bottom... then it is close to the prompt for your next attempt at typing the command correctly.   See: https://github.com/cisagov/development-guide/blob/develop/project_setup/scripts/terraform-to-secrets

👍

Reply:
See: ae21af1